user_sync/connector/directory_csv.py

```diff
@@ -37,12 +37,16 @@ def connector_initialize(options):
     state = CSVDirectoryConnector(options)
     return state
 
-def connector_load_users_and_groups(state, groups):
+def connector_load_users_and_groups(state, groups, extended_attributes):
     '''
     :type state: CSVDirectoryConnector
     :type groups: list(str)
+    :type extended_attributes: list(str)
     :rtype (bool, iterable(dict))
     '''
+    if (extended_attributes is not None):
+        self.logger.warning("CSV directory connector doesn't support extended_attributes; ignored")
```

Comment:
Right, that's exactly my point 👍 .  We are warning the user that the CSV connector doesn't support extended attributes but in fact I believe the CSV connector *already* supports extended attributes without our doing any coding!